sdk/azcore/example_test.go

```diff
@@ -39,12 +41,12 @@ func ExamplePipeline_Do() {
 }
 
 func ExampleRequest_SetBody() {
```

Comment:
Besides the fact the linter is complaining, I wonder if we should remove this example (and the one for Do above).  Customers will never call either of these methods, it's purely codegen.

Reply:
Went ahead and removed these examples.